src/main/kotlin/com/pubnub/api/endpoints/objects/internal/ReturningCollection.kt

```diff
@@ -1,29 +1,12 @@
 package com.pubnub.api.endpoints.objects.internal
 
 import com.pubnub.api.models.consumer.objects.PNPage
-import com.pubnub.api.models.consumer.objects.PNSortKey
+import com.pubnub.api.models.consumer.objects.SortBase
 
 data class ReturningCollection(
     private val limit: Int? = null,
     private val page: PNPage? = null,
     private val filter: String? = null,
-    private val sort: Collection<PNSortKey> = listOf(),
```

Comment:
I like CollectionQueryParameters better than ReturningCollection but can we change it not breaking compatibility?
For now I didn't rename it but I am happy to do this once we agreed it is right thing.

Reply:
In very strict world it would be a compatibility break, but if we assume that the `PubNub` is our interface through which customers are using our library then we can safely rename it since it's not leaking anywhere in the `PubNub` methods, nor we require to create it or pass it anywhere